docs/README.md

```diff
@@ -47,6 +47,28 @@ There are several optional parameters that could be passed into `CreateVolumeReq
 * `gp3` is currently not supported on outposts. Outpost customers need to use a different type for their volumes.
 * Unless explicitly noted, all parameters are case insensitive (e.g. "kmsKeyId", "kmskeyid" and any other combination of upper/lowercase characters can be used).
 
+## Tagging
+To help manage volumes in the aws account, CSI driver will automatically add tags to the volumes it manages.
+
+| TagKey                 | TagValue          | sample                                            | Description         |
+|------------------------|-------------------|---------------------------------------------------|---------------------|
+| CSIVolumeName          | pvcName           | CSIVolumeName = pvc-1234567890                    | add to all volumes, for recording associated pvc id and checking if a given volume was already created so that ControllerPublish/CreateVolume is idempotent. |
+| CSISnapshotName        | snapshotId        | CSISnapshotName = snap-1234567890                 | add to all snapshots, for for recording associated VolumeSnapshot id and checking if a given snapshot was already created                                    |
+| ebs.csi.aws.com/cluster| true              | ebs.csi.aws.com/cluster = true                    | add to all volumes and snapshots, for allowing users to use a policy to limit csi driver's permission to just the resources it manages.                      |
+| kubernetes.io/cluster/X| owned             | kubernetes.io/cluster/aws-cluster-id-1 = owned    | add to all volumes and snapshots if k8s-tag-cluster-id argument is set to X. In in-tree cloudprovider, it's extremely unlikely/impossible for clusterName NOT to be set, so effectively this tag was set on ALL in-tree volumes, [here](https://github.com/kubernetes/cloud-provider-aws/blob/5f394ba297bf280ceb3edfc38922630b4bd83f46/pkg/providers/v2/tags.go#L108). In CSI world we have no automatic way to discover the cloud provider's clusterName, so if the user wants their CSI volumes to have these tags like in-tree volumes would, they have to provide clusterId.|
```

Comment:
you can just keep the first line, the next ones are probably too much detail

Reply:
Gotcha